ui/plugins/auto_classification/main.html

```diff
@@ -39,11 +39,13 @@
           <select
              ng-if="line.type === 'structured' && line.updateText.length > 1"
              ng-model="line.updateSelectOption"
+             title="{{loggedIn() ? '' : 'Must be logged in'}}"
+             ng-disabled="!loggedIn()"
              ng-options="value for (key, value) in line.updateText"
              ></select><br>
           <button class="btn btn-xs btn-default"
-                  title="Select best match and click here"
-                  ng-disabled="!line.canSave"
+                  title="{{loggedIn() ? 'Select best match and click here' : 'Must be logged in'}}"
```

Comment:
I guess we might want to say something if the canSave is false, but the user is not logged in ("No lines to save"?).